docker/kernel/linuxkit/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM linuxkit/kernel:4.9.184 AS ksrc
+FROM alpine:3.4 AS probe-build
+ARG FALCOVER=0.17.0
+ARG SYSDIGVER=0.26.2
+
+COPY --from=ksrc /kernel-dev.tar /
+
+RUN apk add --no-cache --update wget ca-certificates \
```

Comment:
I think there's a way I can refactor this. For instance, we should be able to pull the Falco driver source tree from the falco-minimal image instead of from github. I think this will let us eliminate the need to pull sysdig as well. Also I think we can use DKMS instead of cmake to build the module. 

Let me try to improve this.